dev/com.ibm.ws.cdi.api_fat/test-applications/alterableContextApp.war/src/com/ibm/ws/cdi12/alterablecontext/test/AlterableContextTestServlet.java

```diff
@@ -33,12 +33,14 @@
     private static String SECOND = "Now the command returns: null";
 
     @Test
-    public void testBeanWasFound(HttpServletRequest request, HttpServletResponse response) throws Exception {
+    @Mode(TestMode.FULL)
+    public void testBeanWasFound() throws Exception {
         List<String> strings = DirtySingleton.getStrings();
         assertContains(strings, FIRST);
```

Comment:
`assertThat(strings, hasItem(containsString(FIRST)))`

The current `assertContains` method does not appear to output the actual value.